LABELS.md

```diff
@@ -0,0 +1,322 @@
+# Labels
+
+The following lays out common and recommended labels to use with
+System Containers.
+
+
+## architecture
+
+Describes the architecture of the image.
+
+This label is RECOMMENDED.
+
+### Example
+```
+architecture="x86_64"
+```
+
+
+## atomic.type
+
+Denotes the type of image to the atomic command. If the image is to be used
+as a system container only, then this label should be provided and set to
+system.
+
+This label MUST be used if the image is for use as a system container only.
+
+**Options**: system
+
+### Example
+```
+atomic.type="system"
+```
+
+## authoritative-source-url
+
+The official location the image is published and available.
+
+### Example
+```
+authoritative-source-url="https://example.org/registry/myimage"
+```
+
+
+## build-date
+
+The RFC-3339 date the image was built.
+
+### Example
+```
+build-date="2017-12-01T10:30:30Z"
+```
+
+
+## bzcomponent
```

Comment:
Actually this is com.redhat.component: https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/labels.md